master/internal/resourcemanagers/fair_share.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/determined-ai/determined/master/internal/sproto"
 	"github.com/determined-ai/determined/master/pkg/actor"
 	"github.com/determined-ai/determined/master/pkg/check"
+	"github.com/determined-ai/determined/master/pkg/mmath"
```

Comment:
"mmath" == "more math"

Reply:
[bikeshedding] Maybe `mathx` or `xmath`? I've seen a few uses of the `x` for modified/extended versions of things (`github.com/huandu/xstrings`, `github.com/jmoiron/sqlx`, `github.com/stretchr/objx`).